### ISSUE TITLE
[Bug]: Blocks in doors cannot be rotated

### ISSUE THREAD

**Issue by WhiteBoi1211** [XFactHD/FramedBlocks#413]
### Minecraft version

1.20

### Forge version

47.2.0

### FramedBlocks version

9.2.0

### Describe the bug

It seems impossible to rotate a directional block (Logs etc.) while in a door

Right-clicking with the Screwdriver does nothing, shift right-clicking opens the door.

![image](https://github.com/XFactHD/FramedBlocks/assets/112117824/860350ef-258e-4ab6-b9bd-6c8a72aa25ac)

### How to reproduce the bug

Place any block, which can be rotated in a door, and try to rotate.

### Expected behavior

The texture would change as per any other block

**Comment by XFactHD** [comment]
If you are using any "double door" mod, try removing it (or, if it's Quark, disable the module in their config), this is the most likely cause. If this fixes it, then there is unfortunately nothing I can do as certain implementations of that feature cannot deal with the fact that framed doors have additional interaction behaviors. If it does not fix the issue, then I'll need a full mod list, ideally in the form of a debug crash-report (hold F3+C until the game crashes).

**Comment by WhiteBoi1211** [comment]
I've actually tried this with only FramedBlocks since I don't have any double door mod.

This didn't resolve the issue.

I've updated to the 9.2.1 version of FramedBlocks since your patch note reads:

> Fix camo rotation with the Framed Screwdriver not updating on the client

This actually resolved it, I'm able to rotate all rotatable blocks in doors and regular blocks!

Thanks for looking into this tho! 

Edit:
I should've tried the newest version before making an issue, my bad.
